runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -342,6 +332,14 @@ void* chpl_gpu_impl_memmove(void* dst, const void* src, size_t n) {
   #endif
 }
 
+void* chpl_gpu_impl_memset(void* addr, const uint8_t val, size_t n) {
```

Comment:
We'll need a ROCM version of this as well.

In here (https://github.com/ROCm-Developer-Tools/HIPIFY/blob/master/doc/markdown/CUDA_Driver_API_functions_supported_by_HIP.md) it shows that there is a `hipMemsetD8`. So I think:

`ROCM_CALL(hipMemsetD8((hipDeviceptr_t)addr, (unsigned int)val, n));`